config/dosarray_config.sh

```diff
@@ -34,6 +34,13 @@ then
   exit 1
 fi
 
+# Check if target index is within allowed range
+if [ ${DOSARRAY_TARGET_SERVER_INDEX} -ge  $(( ${#DOSARRAY_PHYSICAL_HOSTS_PUB[@]} - 1 )) ]
+then
+  printf "Check dosarray_config.sh for errors \nDOSARRAY_TARGET_SERVER_INDEX cannot exceed $(( ${#DOSARRAY_PHYSICAL_HOSTS_PUB[@]} - 1 ))"
```

Comment:
This currently outputs to stdout -- please ensure that all error messages go to stderr (` >&2`)